core-rust/state-manager/src/store/rocks_db.rs

```diff
@@ -142,6 +148,7 @@ const ALL_COLUMN_FAMILIES: [RocksDBColumnFamily; 17] = [
     LedgerProofByStateVersion,
     LedgerProofByEpoch,
     Substates,
+    SubstateNodeMetadatas,
```

Comment:
Ditto, `SubstateNodeMetadata`*

Reply:
huh, then it clashes with a struct of the same name :(
I'll figure something out.